src/main/kotlin/step3/view/InputView.kt

```diff
@@ -0,0 +1,30 @@
+package step3.view
+
+class InputView {
+    companion object {
+        private const val CAR_NUMBER_INPUT_MESSAGE = "자동차 대수는 몇 대인가요?"
+        private const val PLAY_TIME_INPUT_MESSAGE = "시도할 횟수는 몇 회인가요?"
+        private const val ERROR = "[ERROR] "
+
+        fun setInputData(): Pair<Int, Int> {
+            return try {
+                val carNumber = inputCarNumber()
+                val playTime = inputPlayTime()
+                Pair(carNumber, playTime)
+            } catch (e: Exception) {
+                println(ERROR + e.message)
+                setInputData()
+            }
+        }
+
+        private fun inputCarNumber(): Int {
+            println(CAR_NUMBER_INPUT_MESSAGE)
+            return readLine()!!.toInt()
```

Comment:
```suggestion
            return readln().toInt()
```
코틀린 버전 1.6 부터 not null 입력값을 받을수 있는 함수가 제공되고 있어요 😄

Reply:
꿀팁 감사합니다 😃 아래와 같이 변경해 보았어요! 😄 

- https://github.com/next-step/kotlin-racingcar/commit/ba00a88bd3d8c3d9cec785c04565f2372735b8f9